### DISCUSSION TITLE
Authorization management not working with mongodb having certification authority

### DISCUSSION THREAD

**Discussion by flower-of-the-bridges** [mia-platform/community#318] [Q&A] [answered]
### Which section?

Console

### Which is the version?

v11.4.2

### Description

Hi

I'm trying to enable authorization on some microservices with rbac policies.

I'm on a console on premise with a MongoDB driver having a certification authority, so in each microservice this certification authority is inserted as a secret to perform successfully the tls handshake and authenticate the connection.

However, the `rbac-service` deployed as sidecar with the microservices for which I've defined a policy does not start, since it has a `MONGODB_URL` environment variable which is unable to fetch the certification authority file.

Looking at the deployment generated by the Console, the secret is only mounted in the main container, but not in the sidecar.

How can I solve this issue?
 

### Actual Outcome

The certification authority is not inserted into the rbac-service, which goes in `CrashLoopbackOff`

### Expected Outcome

 I expected that the `rbac-service` would have the certification authority secret mounted in its file system

**Comment by fredmaggiowski** [comment, marked as answer]
Hi, there is no support for this at the moment, please open a feature request so that we can track it and provide the improvement as soon as possible :)
